app/models/label/LabelTable.scala

```diff
@@ -497,14 +505,15 @@ object LabelTable {
          |       array_to_string(lb_big.tags, ','),
          |       at.low_quality,
          |       at.incomplete,
-         |       at.stale
-         |FROM label AS lb1,
-         |     gsv_data,
-         |     audit_task AS at,
-         |     street_edge_region AS ser,
-         |     sidewalk_user AS u,
-         |     label_point AS lp,
-         |     (
+         |       at.stale,
+         |       comment.comments
+         |FROM label AS lb1
+         |     LEFT JOIN gsv_data ON lb1.gsv_panorama_id = gsv_data.gsv_panorama_id
```

Comment:
@cyrusnaficy Did all of the joins have to be turned into left joins? From a glance, it looks to me like the rest of the query would be able to remain the same, and you would only need to left join with the `validation_task_comment` table?

Reply:
Thanks for pointing this out, I see what you mean. Just pushed a new commit and tested to make sure everything still works fine.

Reply:
A label with no comments would look like this for the JSON response

```
{
    "label_id": 49965,
    "gsv_panorama_id": "mQdJ0civrV0Em2LfNOdN5Q",
    "tutorial": false,
    "image_capture_date": "2018-09",
    "heading": 155.90847778320312,
    "pitch": -6.375,
    "zoom": 1,
    "canvas_x": 238,
    "canvas_y": 379,
    "street_edge_id": 17750,
    "region_id": 30,
    "timestamp": 1556944464421,
    "label_type_key": "SurfaceProblem",
    "label_type_value": "Surface Problem",
    "severity": 4,
    "temporary": false,
    "description": null,
    "user_validation": null,
    "num_agree": 1,
    "num_disagree": 2,
    "num_notsure": 1,
    "comments": null,
    "tags": []
}
```